src/arrayMethodSplice.js

```diff
@@ -4,8 +4,61 @@
  * Implement method Splice
  */
 function applyCustomSplice() {
-  [].__proto__.splice2 = function(start, deleteCount, ...items) {
+  [].__proto__.splice2 = function(
+    startIndex,
+    deleteCount = startIndex > 0 ? (this.length - startIndex) : -startIndex,
+    ...items
+  ) {
     // write code here
+    let deleted = [];
```

Comment:
let's add a noun to the variable name, it's not clear what deleted items should be stored here